src/ProjectNeon/Assets/Core/EventOrchestrator.cs

```diff
@@ -2,6 +2,10 @@
 using System.Linq;
 using UnityEngine;
 
+/**
+ * @todo #190:15min Improve Event Orchestrator documentation. Code is not unsderstandabe by itself, and creating new
```

Comment:
Can you clarify what specifically needs to be added to the documentation?

There are only 2 Editor fields...